### PR TITLE
fix: #id 22271 Take out depreciated toolbar config

### PR DESCRIPTION
**Pull request by zona-ciq**
fix: #id 22271

[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/22271/details/)

**Description of change**
* Took out depreciated toolbar config
* Set the "autoShow" flag inside the toolbar config to be true in case user set the workspaceService.global to be false (currently undocumented). In that case if we have autoShow set to be false, the toolbar would come up hidden.

**Description of testing**
Markdown will convert the 1s to the appropriate number.
1. Launch finsemble
1. Make sure toolbar shows up and cannot tab, tile, nor snap

### CODE REVIEW

File: configs/application/UIComponents.json [hunk 1]
```diff
@@ -595,7 +595,7 @@
 				"docked": "top",
 				"canGroup": false,
 				"options": {
-					"autoShow": false,
+					"autoShow": true,
```
Comment (carterbs) [inline configs/application/UIComponents.json]: @zona-ciq why did you change this again? It should probably be in the PR description.

**Comment by carterbs** [comment]
@zona-ciq how does changing autoshow impact startup appearance? IIRC we had the toolbar hidden because it comes up, figures out where it needs to go, and then moves itself. That, and we didn't want a white toolbar on startup.

File: configs/application/UIComponents.json (hunk 1 above)
Comment (zona-ciq) [reply configs/application/UIComponents.json]: I just tested with workspaceService.global set to true and toolbar did come up initially blank. I'll change it back for now. When we document the `global` attribute we need to be aware that if the `autoShow` field is set to "false" the toolbar wouldn't automatically show up on startup.
